src/activity_actor.cpp

```diff
@@ -455,10 +455,11 @@ void dig_activity_actor::finish( player_activity &act, Character &who )
                                   calendar::turn ) );
     }
 
-    const int helpersize = character_funcs::get_crafting_helpers( who, 3 ).size();
-    who.mod_stored_nutr( 5 - helpersize );
-    who.mod_thirst( 5 - helpersize );
-    who.mod_fatigue( 10 - ( helpersize * 2 ) );
+    int act_exertion = act.moves_total;
```

Comment:
```suggestion
    const int act_exertion = act.moves_total;
```
To make the code easier to understand, it'd be great if all variable are declared immutable (const) unless there's a compelling reason to mutate it.